tests/web/test_websocket_server.py

```diff
@@ -154,15 +156,16 @@ def test_websocket_string_event(test_client, loop):
 
 
 @asyncio.coroutine
-def test_websocket_block_event(b, test_client, loop):
+@pytest.mark.parametrize('_block', (10,), indirect=('_block',), ids=('block',))
+def test_websocket_block_event(b, _block, test_client, loop):
     from bigchaindb import events
     from bigchaindb.web.websocket_server import init_app, POISON_PILL, EVENTS_ENDPOINT
 
     event_source = asyncio.Queue(loop=loop)
     app = init_app(event_source, loop=loop)
     client = yield from test_client(app)
     ws = yield from client.ws_connect(EVENTS_ENDPOINT)
-    block = create_block(b, 10).to_dict()
+    block = _block.to_dict()
```

Comment:
OK but I believe the previous solution was more understandable[:](https://twitter.com/vrde/status/852075949835382784)

```python
def create_block(b, total=1):
    # ...


def test_websocket_block_event(b, test_client, loop):
    # ...
    block = create_block(b, 10)
    # ...
```

vs

```python
@pytest.fixture
def _block(b, request):
    total = getattr(request, 'param', 1)
    # ...


@pytest.mark.parametrize('_block', (10,), indirect=('_block',), ids=('block',))
def test_websocket_block_event(b, _block, test_client, loop):
    # ...
    block = _block.to_dict()
    # ...
```